src/AppBundle/EventListener/RedirectToPreferredLocaleSubscriber.php

```diff
@@ -23,7 +25,7 @@
  *
  * @author Oleg Voronkovich <oleg-voronkovich@yandex.ru>
  */
-class RedirectToPreferredLocaleListener
+class RedirectToPreferredLocaleSubscriber implements EventSubscriberInterface
```

Comment:
Now that event subscribers can be made private, the new best practice should be to always use them in place of listeners. Thus, I think also that *not* advertising listeners in symfony-demo is a good thing - even if it works the same as before.